src/GraphQl/Type/TypeBuilderInterface.php

```diff
@@ -43,7 +43,7 @@ public function getNodeInterface(): InterfaceType;
     /**
      * Gets the type of a paginated collection of the given resource type.
      */
-    public function getResourcePaginatedCollectionType(GraphQLType $resourceType, Operation $operation): GraphQLType;
+    public function getResourcePaginatedCollectionType(GraphQLType $resourceType, string $resourceClass, Operation $operation): GraphQLType;
```

Comment:
Couldn't we find a way to keep this change?

Reply:
lets keep this right now to not break the interface? Then we need to deprecate using this argument and offer maybe a new interface?